cmd/glasskube/cmd/bootstrap.go

```diff
@@ -31,6 +33,19 @@ var bootstrapCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
 		client := bootstrap.NewBootstrapClient(cfg)
+		installedVersion, err := clientutils.GetPackageOperatorVersion(cmd.Context())
+		if err != nil {
+			fmt.Printf("error : %v\n", err)
```

Comment:
This is problematic, because if the package operator is not installed yet (i.e. glasskube is not bootstrapped), then the user would always run into this error and could never bootstrap. 

I suggest that in the case that `err != nil`, you additionally call `bootstrap.IsBootstrapped`: If the result is true, you can actually print the error (because the operator should be there, but isn't for some reason). If the result is false, continue with the regular bootstrap flow that starts here in line 49.

Reply:
Does this look good:
```
IsBootstrapped, err := bootstrap.IsBootstrapped(cmd.Context(), cfg)
		if err != nil && !IsBootstrapped {
			fmt.Printf("error : %v\n", err)
			return
		}
```

Reply:
@hanshal101 please commit your current status to the branch so I can see the whole context.

Reply:
@hanshal101 I see, my idea was to do the bootstrap check only if `err != nil` after `GetPackageOperatorVersion()`.